shell/probe/demo.sh

```diff
@@ -1,5 +1,9 @@
 #!/bin/bash
 
+GREEN='\033[1;32m'
```

Comment:
how's this green look on your ultra green terminal?

Reply:
lol now some of the green is bold, can you tell?

![Screen Shot 2023-01-10 at 5 30 30 AM](https://user-images.githubusercontent.com/9056078/211564476-0ccea336-2c29-4d12-a6a0-08a517327f51.png)